### PR TITLE
Reduce icud.dll

### PR DESCRIPTION
**Pull request by mryzhov**
Addressed issue  146494

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,37 @@ After that you can add binary extension in the code with:
...
+      "localeFilter": {
+        "filterType": "language",
+        "includelist": [
+          "en"
+        ]
+      }
+    }
+    ```
+
+By following these instructions, you can effectively reduce the size of the ICU libraries in your final package.
```
Comment (Wovchena) [inline README.md:167]: What are the drawbacks? It should be enabled by default or explained why one may not want to do that.

Comment (mryzhov) [reply README.md:167]: by default ICU include all supported languages, this config allows to exclude unnecessary languages and features.

**Review by apaniukov — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Wovchena) [reply README.md:167]: I understand that. Why don't tokenizers excluded them by default?

Comment (mryzhov) [reply README.md:167]: To support all possible languages by default

Comment (mryzhov) [reply README.md:173]: updated readme

**Review by Wovchena — APPROVED** [review]

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -47,6 +52,13 @@ endif()
 
 option(BUILD_FAST_TOKENIZERS "Compile core_tokenizers instead of downloading prebuilt library. The option is forced to ON if _GLIBCXX_USE_CXX11_ABI=0" ${USE_ABI0})
 
+option(ICU_DATA_FILTER_FILE "Path to the custom data filter file for ICU4C data packaging" "")
+if(ICU_DATA_FILTER_FILE)
+  set(ENV{ICU_DATA_FILTER_FILE} "${ICU_DATA_FILTER_FILE}")
+endif()
+
+message(STATUS "ICU_DATA_FILTER_FILE is $ENV{ICU_DATA_FILTER_FILE}")
+
```
Comment (mryzhov) [inline src/CMakeLists.txt:61]:
```suggestion
```

**Pull request merged**